pkg/p2p/libp2p/libp2p.go

```diff
@@ -53,6 +54,21 @@ import (
 	"go.uber.org/atomic"
 )
 
+func init() {
+	// override the libp2p defaults
+	rcmgr.DefaultLimits.SystemBaseLimit.StreamsInbound = 8096
+	rcmgr.DefaultLimits.SystemMemory.MemoryFraction = 0.3
+	rcmgr.DefaultLimits.TransientBaseLimit.StreamsInbound = 1024
+	rcmgr.DefaultLimits.TransientBaseLimit.StreamsOutbound = 2024
+	rcmgr.DefaultLimits.TransientBaseLimit.Streams = 2048
+	rcmgr.DefaultLimits.ServiceBaseLimit.StreamsInbound = 4096
+	rcmgr.DefaultLimits.ServicePeerBaseLimit.StreamsInbound = 1024
```

Comment:
i would bump ServicePeerBaseLimit.StreamsInbound to keep the ratio between the two the same